src/pocketmine/Player.php

```diff
@@ -3416,10 +3416,19 @@ public function kick($reason = "", $isAdmin = true){
 	public function addTitle(string $title, string $subtitle = "", int $fadeIn = -1, int $stay = -1, int $fadeOut = -1){
 		$this->setTitleDuration($fadeIn, $stay, $fadeOut);
 		if($subtitle !== ""){
-			$this->sendTitleText($subtitle, SetTitlePacket::TYPE_SET_SUBTITLE);
+			$this->addSubTitle($subtitle);
 		}
 		$this->sendTitleText($title, SetTitlePacket::TYPE_SET_TITLE);
 	}
+	
+	/**
+	 * Sets the subtitle message, without a title.
+	 *
+	 * @param strint $subtitle
+	 */
+	public function addSubTitle(string $subtitle){
```

Comment:
It has to be a seperate method because i called addTitle( "" , "subtitle"); and it showed a blank title with the subtitle, which shouldnt happen